### PR TITLE
Change release title; expose number of points on the draw_random_points function

### PR DESCRIPTION
**Pull request by brunodrugowick**
I thought this was an issue with the token, but I use the same token [here](https://github.com/brunodrugowick/timeseries-poc/blob/main/.github/workflows/release.yml#L39) and it works just fine.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           tag_name: "${{ steps.read-version.outputs.version }}"
-          release_name: Blood Pressure Diary v${{ steps.read-version.outputs.version }}
+          release_name: The Chaos Game (Visualization) v${{ steps.read-version.outputs.version }}
```
Comment (brunodrugowick) [inline .github/workflows/release.yml:45]: Copy & Pasta mistake. 😬

**Comment by brunodrugowick** [comment]
Would you kindly check this configuration in the repo?

image

**Comment by brunodrugowick** [comment]
> Would you kindly check this configuration in the repo?
> 
> image

Well, if this was the problem, the changelog action would also fail. :(

I guess if we don't figure this out, let's just remove this `release.yml` workflow.

**Comment by brunodrugowick** [comment]
@adalbertogonzaga it might be that I don't have permission to create releases in this repo, and the merge action comes from my user, because I'm the author of the commit that merges into `main`.

To test/confirm this theory we would need a PR from you merged to `main`. You can make a PR with the same fix I have on this one and merge. 🤷🏼

File: pygame_start.py
Comment (brunodrugowick) [inline pygame_start.py:1]: I'm sneaking in some changes to this file. 😄

**Comment by brunodrugowick** [comment]
GOOD LUCK WITH ALL THE EMAILS I'M TRIGGERING TO YOU! MUAHAHAHAHA

image

**Comment by brunodrugowick** [comment]
Ahá! This might be it. In the Repo Settings > Actions > General: 

image

File: pygame_start.py [hunk 2]
```diff
@@ -67,14 +72,14 @@ def draw_score(screen, score):
     screen.blit(score_surface, (width - score_surface_width, 5))
 
 
-def draw_random_points(screen, click_position):
+def draw_random_points(screen, click_position, points):
```
Comment (brunodrugowick) [inline pygame_start.py:75]: Exposing `points` allows us to config this dynamically in the future.

**Comment by adalbertogonzaga** [comment]
> Would you kindly check this configuration in the repo?
> 
> image

The configurations for the actions section is already checked and it is the same you've indicated. I believe it could be something related to the repo security... I've activated some stuff to prevent commits on main and PR without revisions...

**Review by adalbertogonzaga — APPROVED** [review]
I believe in it

**Comment by brunodrugowick** [comment]
> I believe in it

Check the config before I (or you) merge.

**Comment by adalbertogonzaga** [comment]
> Ahá! This might be it. In the Repo Settings > Actions > General:
> 
> image

Done. Let me know if anything weird occurs

**Pull request merged**